app-modules/ai/database/factories/PromptFactory.php

```diff
@@ -55,6 +55,7 @@ public function definition(): array
             'description' => fake()->optional()->sentences(asText: true),
             'prompt' => fake()->sentences(asText: true),
             'type_id' => PromptType::query()->inRandomOrder()->first() ?? PromptType::factory()->create(),
+            'is_smart' => fake()->boolean(),
```

Comment:
You are getting random failures in other tests because you have now added `is_smart` as a random value into this factory. So the tests that are failing are most likely assuming that it is the default of `false`.

So you need to go to those tests and where it make sense fix them to be false. Or change this to a state and apply it where it is needed.